### PR TITLE
Engine: increase memory for large transaction test

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**
This PR increase memory limit for large transaction engine test in order to reduce GC overhead.

## Pull Request Checklist

- [X] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Comment by SamirTalwar-DA** [comment]
What's with all the TypeScript changes?

**Comment by remyhaemmerle-da** [comment]
> What's with all the TypeScript changes?

rebase issue.

**Comment by cocreature** [comment]
Did we actually see OOMs or is this trying to fix the slowdown caused by different GC behavior due to lower memory limits?

**Comment by remyhaemmerle-da** [comment]
> Did we actually see OOMs or is this trying to fix the slowdown caused by different GC behavior due to lower memory limits?
This is two limit the slowdown caused by GC behaviour.

> Did we actually see OOMs or is this trying to fix the slowdown caused by different GC behavior due to lower memory limits?

This is to fix slowdown caused by GC.

File: daml-lf/engine/BUILD.bazel [hunk 1]
```diff
@@ -56,3 +58,31 @@ da_scala_test(
...
+    name = "test-large-transaction",
+    timeout = "moderate",
+    srcs = glob([
+        "src/test/**/LargeTransactionTest.scala",
+        "src/test/**/InMemoryPrivateLedgerData.scala",
+    ]),
+    data = [
+        "//daml-lf/tests:LargeTransaction.dar",
+    ],
+    initial_heap_size = "2g",
```
Comment (cocreature) [inline daml-lf/engine/BUILD.bazel:72]: Can we add a comment here that this is for performance reasons?

**Pull request merged**